anomalib/models/components/freia/modules/all_in_one_block.py

```diff
@@ -0,0 +1,289 @@
+"""All in One Block Module."""
+
+# Copyright (c) 2018-2022 Lynton Ardizzone, Visual Learning Lab Heidelberg.
+# SPDX-License-Identifier: MIT
+#
```

Comment:
Sorry, one more question: the file contains only one FrEIA copyright, did you make change in some of the FrEIA files?

If yes, then in the files add 2-line with Intel copyright too.
If no, then the original FrEIA copyright only is OK.

Reply:
I've changed quite minor stuff, just minor typos and additions in docstring.